crescent/ext/tasks/cron.py

```diff
@@ -10,24 +10,41 @@
 
 
 class Cronjob(Task):
-    def __init__(self, cron: str, callback: TaskCallbackT) -> None:
+    def __init__(self, cron: str, callback: TaskCallbackT, *, on_startup: bool) -> None:
         self.cron: croniter = croniter(cron, datetime.now())
+        self.on_startup = on_startup
```

Comment:
This value is behaving more like a tracker. I would name it "must_call_first_iteration" so you can see it's not a constant value.

Reply:
how about `call_first_iteration`?

Reply:
> Why not initialize?

i dont think that makes sense here

Reply:
more like `on_init`. but whatever